src/appMain/sdl_preloaded_pt.json

```diff
@@ -2276,13 +2356,29 @@
                 "steal_focus": false,
                 "priority": "NONE",
                 "default_hmi": "NONE",
-                "groups": ["Base-4"]
+                "groups": ["Base-4"],
+                "groups_nonPrimaryRC": [

```

Comment:
@yang1070 
Some parts have been left to consider logic extension in future as this implementation is assumed as first part. Do you want to remove that?